musmart/melody/contour/step_contour.py

```diff
@@ -0,0 +1,217 @@
+"""Calculates the Step Contour of a melody, along with related features, as implemented
```

Comment:
The package has now been reorganised so that musmart is called amads. When you are ready to do so, you should merge master into your branch and then move the file accordingly. I would probably wait until you have addressed the other comments though as moving the file might mess up the comment alignment.